docs/content/en/docs/packages/packagecontroller.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Managing the package controller"
+linkTitle: "Package controller"
+weight: 4
+aliases:
+    /docs/tasks/packages/packagecontroller/
+---
+
+### Installing the package controller
+
+{{% alert title="Important" color="warning" %}}
+The package controller installation creates a package bundle controller resource for each cluster, thus allowing each to activate different package bundle version. Ideally, you should never delete this resource because it would mean losing that information and upon re-installing, the latest bundle will be selected. However, you can always go back to the previous bundle version. For more information, see [Managing package bundles.]({{< relref "./packagebundles" >}})
```

Comment:
```suggestion
The package controller installation creates a package bundle controller resource for each cluster, thus allowing each to activate a different package bundle version. Ideally, you should never delete this resource because it would mean losing that information and upon re-installing, the latest bundle would be selected. However, you can always go back to the previous bundle version. For more information, see [Managing package bundles.]({{< relref "./packagebundles" >}})
```